tools/get.py

```diff
@@ -71,13 +71,13 @@ def get_tool(tool):
     local_path = dist_dir + archive_name
     url = tool['url']
     #real_hash = tool['checksum'].split(':')[1]
-    if 'CYGWIN_NT' in sys_name:
+    if 'CYGWIN_NT' not in sys_name:
```

Comment:
but this will disable SSL check on all other OSs. Maybe you need to install something to get it going?